### PR TITLE
New selfmade keyboard

### PR DESCRIPTION
**Pull request by Pascal0816**
I created a new keyboard for myself, that is a 96%ish keyboard with a full-size numberpad.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x] Keyboard (addition or update)
- [ x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ]x I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ingenieurrr/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,     _______,
+        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,     _______,
+        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                _______,    _______,    _______,    _______,
+        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                _______,    _______,    _______,    _______,     _______,
+        _______,    _______,    _______,                                        _______,                                        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______
```
Comment (waffle87) [inline keyboards/ingenieurrr/keymaps/default/keymap.c:22]:
```suggestion
        KC_LCTL,    KC_LGUI,    KC_LALT,                                        KC_SPC,                                         KC_RALT,    _______,      KC_RCTL,    KC_LEFT,    KC_DOWN,    KC_RGHT,    KC_P0,      KC_PDOT
```

File: keyboards/ingenieurrr/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,     _______,
+        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,     _______,
+        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                _______,    _______,    _______,    _______,
+        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                _______,    _______,    _______,    _______,     _______,
+        _______,    _______,    _______,                                        _______,                                        _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______
```
Comment (waffle87) [inline keyboards/ingenieurrr/keymaps/via/keymap.c:21]:
```suggestion
        KC_LCTL,    KC_LGUI,    KC_LALT,                                        KC_SPC,                                         KC_RALT,    _______,      KC_RCTL,    KC_LEFT,    KC_DOWN,    KC_RGHT,    KC_P0,      KC_PDOT
```

File: keyboards/ingenieurrr/readme.md
Comment (waffle87) [inline keyboards/ingenieurrr/readme.md:1]: This file must follow the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md).

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/ingenieurrr/info.json
Comment (drashna) [inline keyboards/ingenieurrr/info.json:1]: This file has a mix of spaces and tabs.  Could you run `qmk format-json` on it?

**Pull request closed without merging**